src/style/apply.js

```diff
@@ -206,6 +206,13 @@ styfn.applyContextStyle = function( cxtMeta, cxtStyle, ele ){
     // save cycles when the context prop doesn't need to be applied
     if( eleProp === cxtProp ){ continue; }
 
+    // save cycles when a mapped context prop doesn't need to be applied
+    if( is.fn( cxtProp.value )) {
+      cxtProp.fnValue = cxtProp.value( ele ); // temporarily cache the value in case of a miss
+
+      if( cxtProp.fnValue === eleProp.previousValue ){ continue; }
+    }
+
```

Comment:
If we use this approach, we must delete `ele.updateMappers()` and `style.updateMappers()`.

Reply:
I'm afraid I'm not too familiar with these functions, what makes them incompatible with this approach?